### ISSUE TITLE
Game trailer

### ISSUE THREAD

**Issue by finnbear** [SoftbearStudios/mk48#86]
The game needs a good trailer to post on video sharing platforms.

- Use `shift + v` to start recording in game, without the overlays, and `v` again to stop
- The trailer will be composed by multiple clips, each demonstrating some cool feature, or a single clip demonstrating them all
- The trailer should not include player names that are overly personal

**Comment by JoeyMeepLand** [comment]
I could help with this i just need to be tault how to do this its not working

**Comment by caibear** [comment]
Yeah the recording feature was removed as it caused bad performance on certain browsers (even after the recording was ended).

Also we need an option to turn of the UI and possibly player names in order to capture high quality clips.

**Comment by JoeyMeepLand** [comment]
good it

**Comment by JoeyMeepLand** [comment]
got it

**Comment by finnbear** [comment]
Next update, there will be a cinematic mode in settings.

Added by https://github.com/SoftbearStudios/mk48/commit/5bf9395f731de60e50bd217026076e5ee3d56919

**Comment by JoeyMeepLand** [comment]
Finn Bear i have a question 1 how do i access all the dev testing stuff on
chromebook 2 do you want me to make you a proper logo or do we 3 do you
want me to make merchandise (like tshirts logos)

**Comment by finnbear** [comment]
Good questions! :)

> i have a question 1 how do i access all the dev testing stuff on chromebook

What makes you think there is dev testing stuff? The only two testing-related things that exist are:
- Right click + Inspect element, which is built into most browsers (unless it is blocked for some reason, such as if you have a school-issued Chromebook). This would just allow you to see low-level network traffic but, since the server enforces everything, this does not allow you much testing flexibility (you can't, for example, spawn an arbitrary ship).
- Debug mode, which simply starts every player/bot with enough points to upgrade to max level. This cannot be accessed on Chromebook, because it requires compiling and hosting the client and server yourself (which is complicated, but instructions are in the README files). Due to an incident when I accidentally deployed a "free 1000 points" feature on the main server, it is now impossible to access it on the main server i.e. outside of debug mode.

> 2 do you want me to make you a proper logo

If you have a logo idea, a sketch would be cool. The game doesn't necessarily need a logo, though.

> or do we 3 do you want me to make merchandise (like tshirts logos)

I personally think it's a bit early for merchandise but, seeing as all the game assets are open source, you can certainly make something and distribute it yourself via a website like https://www.zazzle.com/ (I'd be very curious if Mk48.io merchandise would be successful)
